testlapack/general.go

```diff
@@ -17,8 +17,14 @@ import (
 	"github.com/gonum/lapack"
 )
 
-// dlamchE is the machine epsilon. For IEEE this is 2^-53.
-const dlamchE = 1.0 / (1 << 53)
+const (
+	// dlamchE is the machine epsilon. For IEEE this is 2^-53.
+	dlamchE = 1.0 / (1 << 53)
+	dlamchP = 2 * dlamchE
+	dlamchB = 2
+)
+
+var dlamchS = math.Nextafter((4 / math.MaxFloat64), 0)
```

Comment:
I'd rather not, though that number can be expressed (horribly) as `1.0 / (1 << 256) / (1 << 256) / (1 << 256) / (1 << 254)`. So maybe use that, with the comment. At least it's easy to add the numbers together and confirm that they are 1022. Then that const can go in the block above.